package.json

```diff
@@ -15,14 +15,14 @@
     "cacheDirectory": "./.jest/cache"
   },
   "dependencies": {
-    "jest-snapshot": "^26.3.0"
+    "jest-snapshot": "^27.0.2"
```

Comment:
should the peerDependencies also be changed then ?

Reply:
definitely should, yeah!
Regardless this will be a technically breaking change due to the major bump

Reply:
yep 👍 , I will bump the major version